src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -27,7 +27,7 @@
        To deal with that, we write this target which finds that added package, removes it from
        the references list, then adds it back in again with the 'EmbedInteropTypes' flag properly
        set. -->
-  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolveNuGetPackageAssets">
+  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolvePackageDependenciesForBuild">
     <ItemGroup>
       <_NuGetVisualStudioPackageReference Include="@(Reference)" Condition="'%(Reference.NuGetPackageId)' == 'NuGet.VisualStudio'" />
```

Comment:
Sigh I missed this in the email last night. The 1.1 SDK doesn't have the NuGetPackageId metadata anymore. We'll need to fix this conditional. 

@jasonmalinowski